i3-gaps and python39Packages.powerline are failing. i3-gaps also switched to the new build system. I already have i3-gaps working with 4.19. Should I do a separate PR how to proceed?
```
$ nix build --no-link --keep-going --option build-use-sandbox relaxed -f /home/marcel/.cache/nixpkgs-review/pr-109106-2/build.nix
warning: ignoring the user-specified setting 'sandbox', because it is a restricted setting and you are not a trusted user
builder for '/nix/store/j6irxp0p0f26p8al5f1pvy344kk01abx-i3-gaps-4.18.3.drv' failed with exit code 1; last 10 log lines:
  autoreconf: running: /nix/store/var821ibfzbhjk911z4b1h5xhxc3pfl5-autoconf-2.70/bin/autoheader --force
  autoreconf: running: automake --add-missing --copy --force-missing
  Makefile.am:229: warning: '%'-style pattern rules are a GNU make extension
  autoreconf: './config.sub' is updated
  autoreconf: './config.guess' is updated
  autoreconf: Leaving directory '.'
  configuring
  meson flags: --buildtype=plain         --libdir=/nix/store/41wh68b6sqgy53kql7937yxyaa5y18l2-i3-gaps-4.18.3/lib --libexecdir=/nix/store/41wh68b6sqgy53kql7937yxyaa5y18l2-i3-gaps-4.18.3/libexec         --bindir=/nix/store/41wh68b6sqgy53kql7937yxyaa5y18l2-i3-gaps-4.18.3/bin --sbindir=/nix/store/41wh68b6sqgy53kql7937yxyaa5y18l2-i3-gaps-4.18.3/sbin         --includedir=/nix/store/41wh68b6sqgy53kql7937yxyaa5y18l2-i3-gaps-4.18.3/include         --mandir=/nix/store/41wh68b6sqgy53kql7937yxyaa5y18l2-i3-gaps-4.18.3/share/man --infodir=/nix/store/41wh68b6sqgy53kql7937yxyaa5y18l2-i3-gaps-4.18.3/share/info         --localedir=/nix/store/41wh68b6sqgy53kql7937yxyaa5y18l2-i3-gaps-4.18.3/share/locale         -Dauto_features=enabled         -Dwrap_mode=nodownload         --prefix=/nix/store/41wh68b6sqgy53kql7937yxyaa5y18l2-i3-gaps-4.18.3  
  
  ERROR: Neither directory contains a build file meson.build.
builder for '/nix/store/f8gfdpqmjvb5lbf3wm824jszh6fc2iq1-python3.9-pyuv-1.2.0.drv' failed with exit code 1; last 10 log lines:
  src/pyuv.c:72:5: warning: ‘PyEval_InitThreads’ is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
     72 |     PyEval_InitThreads();
        |     ^~~~~~~~~~~~~~~~~~
  In file included from /nix/store/3g9596lia3mdr753amydqs6ilr4iaf1m-python3-3.9.1/include/python3.9/Python.h:145,
                   from src/pyuv.h:7,
                   from src/pyuv.c:2:
  /nix/store/3g9596lia3mdr753amydqs6ilr4iaf1m-python3-3.9.1/include/python3.9/ceval.h:130:37: note: declared here
    130 | Py_DEPRECATED(3.9) PyAPI_FUNC(void) PyEval_InitThreads(void);
        |                                     ^~~~~~~~~~~~~~~~~~
  error: command '/nix/store/d2fqghv083a8q7j8sjb2m5q6i840h691-gcc-wrapper-10.2.0/bin/gcc' failed with exit code 1
cannot build derivation '/nix/store/cyijpz2r2kpb1qv0ccamzs6shgjlgji3-python3.9-powerline-2.8.1.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/fb4dn34ik8bxsb6a1nql54r7rz29dg6l-env.drv': 2 dependencies couldn't be built
[0 built (2 failed), 0.0 MiB DL]
error: build of '/nix/store/fb4dn34ik8bxsb6a1nql54r7rz29dg6l-env.drv' failed
https://github.com/NixOS/nixpkgs/pull/109106
2 packages failed to build:
i3-gaps python39Packages.powerline

20 packages built:
autotiling flashfocus i3 i3-balance-workspace i3-layout-manager i3-resurrect i3-wk-switch i3altlayout polybarFull powerline python37Packages.i3ipc python37Packages.powerline python37Packages.py3status python38Packages.i3ipc python38Packages.py3status python39Packages.i3ipc python39Packages.py3status raiseorlaunch sway-contrib.inactive-windows-transparency xkb-switch-i3
```

--- a/pkgs/applications/window-managers/i3/default.nix
+++ b/pkgs/applications/window-managers/i3/default.nix
@@ -1,18 +1,18 @@
-{ fetchurl, lib, stdenv, which, pkgconfig, makeWrapper, installShellFiles, libxcb, xcbutilkeysyms
+{ fetchurl, lib, stdenv, pkg-config, makeWrapper, meson, ninja, installShellFiles, libxcb, xcbutilkeysyms
 , xcbutil, xcbutilwm, xcbutilxrm, libstartup_notification, libX11, pcre, libev
 , yajl, xcb-util-cursor, perl, pango, perlPackages, libxkbcommon
 , xorgserver, xvfb_run }:
 
 stdenv.mkDerivation rec {
   pname = "i3";
-  version = "4.18.3";
+  version = "4.19";
 
   src = fetchurl {
-    url = "https://i3wm.org/downloads/${pname}-${version}.tar.bz2";
-    sha256 = "03dijnwv2n8ak9jq59fhq0rc80m5wjc9d54fslqaivnnz81pkbjk";
+    url = "https://i3wm.org/downloads/${pname}-${version}.tar.xz";
+    sha256 = "0wjq6lkidg0g474xsln1fhbxci7zclq3748sda10f1n7q01qp95c";
   };
 
-  nativeBuildInputs = [ which pkgconfig makeWrapper installShellFiles ];
+  nativeBuildInputs = [ pkg-config makeWrapper meson ninja installShellFiles ];
 
   buildInputs = [
     libxcb xcbutilkeysyms xcbutil xcbutilwm xcbutilxrm libxkbcommon

--- a/pkgs/applications/window-managers/i3/gaps.nix
+++ b/pkgs/applications/window-managers/i3/gaps.nix
@@ -3,17 +3,12 @@
 i3.overrideAttrs (oldAttrs : rec {
 
   name = "i3-gaps-${version}";
-  version = "4.18.3";
+  version = "4.19";
 
   src = fetchurl {
-    url = "https://github.com/Airblader/i3/releases/download/${version}/i3-${version}.tar.bz2";
-    sha256 = "1hcakwyz78lgp8mhqv7pw86jlb3m415pfql1q19rkijnhm3fn3ci";
+    url = "https://github.com/Airblader/i3/releases/download/${version}/i3-${version}.tar.xz";
+    sha256 = "0j19kj05fpjfnj50vyykk6jsr07hq9l26y8na55bb7yfrra8yp4h";
   };
-
-  nativeBuildInputs = oldAttrs.nativeBuildInputs ++ [ autoreconfHook ];
-
-  # fatal error: GENERATED_config_enums.h: No such file or directory
-  enableParallelBuilding = false;
 
   meta = with lib; {
     description = "A fork of the i3 tiling window manager with some additional features";

